go.mod

```diff
@@ -9,6 +9,7 @@ require (
 	github.com/go-errors/errors v1.4.0
 	github.com/go-test/deep v1.0.7 // indirect
 	github.com/hashicorp/go-hclog v0.16.2 // indirect
+	github.com/hashicorp/go-plugin v1.4.0 // indirect
```

Comment:
Why do we have changes here? Did you include some new module earlier?

Reply:
@sumovishal , didnt add anything manually. as part of go run , these are included.

Reply:
you can go ahead with the merge. We are just upgrading to a new version here. Should be fine.